pkg/validate/raw/validate/management_annotation_test.go

```diff
@@ -1,64 +0,0 @@
-// Copyright 2022 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package validate
-
-import (
-	"errors"
-	"testing"
-
-	"kpt.dev/configsync/pkg/core"
-	"kpt.dev/configsync/pkg/core/k8sobjects"
-	"kpt.dev/configsync/pkg/importer/analyzer/ast"
-	"kpt.dev/configsync/pkg/importer/analyzer/validation/nonhierarchical"
-	"kpt.dev/configsync/pkg/metadata"
-	"kpt.dev/configsync/pkg/status"
-	"kpt.dev/configsync/pkg/syncer/syncertest"
-)
-
-func TestValidManagementAnnotation(t *testing.T) {
-	testCases := []struct {
-		name string
-		obj  ast.FileObject
-		want status.Error
-	}{
-		{
-			name: "no management annotation",
-			obj:  k8sobjects.Role(),
-		},
-		{
-			name: "disabled management passes",
-			obj:  k8sobjects.Role(syncertest.ManagementDisabled),
-		},
-		{
-			name: "enabled management fails",
-			obj:  k8sobjects.Role(syncertest.ManagementEnabled),
-			want: status.FakeError(nonhierarchical.IllegalManagementAnnotationErrorCode),
-		},
-		{
-			name: "invalid management fails",
-			obj:  k8sobjects.Role(core.Annotation(metadata.ResourceManagementKey, "invalid")),
-			want: status.FakeError(nonhierarchical.IllegalManagementAnnotationErrorCode),
-		},
-	}
-
-	for _, tc := range testCases {
-		t.Run(tc.name, func(t *testing.T) {
-			err := nonhierarchical.ValidManagementAnnotation(tc.obj)
```

Comment:
It seems like this test should be verifying `ManagementAnnotation` in the adjacent file

Reply:
reverted and refactored